### PR TITLE
Add cheesecake recipe

### PR DESCRIPTION
**Pull request by ScratchX98**

### CODE REVIEW

File: src/cheesecake.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- Difficulty: Medium
+
+## Ingredients
+
+For the crust:
+- 85g normal butter
+- 140g digestive biscuits, crumbled
+- 1tbsp sugar, preferably golden caster
+For the cheesecake:
+- 900g Philadelphia cheese, or any other soft cheese
```
Comment (hallzy) [inline src/cheesecake.md:19]:
```suggestion
- 1tbsp sugar, preferably golden caster

For the cheesecake:
- 900g Philadelphia cheese, or any other soft cheese
```

Without an extra line the `For the cheescake:` line actually ends up on the end of the previous line when rendered.

File: src/cheesecake.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+- 250g golden caster sugar
+- 3tbsp plain flour
+- 1.5 tsp vanilla extract
+- 2 tsp grated lemon zest
+- 1.5 tsp lemon juice
+- 3 large eggs, and 1 yolk
+- 285ml sour cream
+For the topping:
+- whatever you want really, I used rasberry jam
```
Comment (hallzy) [inline src/cheesecake.md:28]:
```suggestion
- 285ml sour cream

For the topping:
- whatever you want really, I used raspberry jam
```

File: src/cheesecake.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# Cheesecake
+
+Sweet dish, pretty creamy, you can use whatever topping you like
+
+![cheesecake](pix/cheesecake.webp)
+
+- Prep time: 25 minutes
```
Comment (zyansheep) [inline src/cheesecake.md:7]: Add emojis maybe?

**Review by zyansheep — APPROVED** [review]

File: src/cheesecake.md (hunk 3 above)
Comment (ScratchX98) [reply src/cheesecake.md:7]: alright

**Review by hallzy — APPROVED** [review]

**Pull request closed without merging**